datamapper-e2e/src/test/resources/atlas-itests.properties

```diff
@@ -0,0 +1,8 @@
+http.host=localhost
+http.port=8586
+https.host=localhost
+https.port=8443
+https.keystore=ssl.keystore
+https.keystore.password=atlasmap
+spring.resources.static-locations=/Users/mmelko/RH_WORK/Work/Test/syndesis/atlasmap-qe/datamapper-e2e/src/main/resources/static/
```

Comment:
This path is probably not very generic :)

Reply:
it's even not used :)